app/src/main/java/Activities/ViewRecordActivity.java

```diff
@@ -47,12 +58,14 @@ protected void onCreate(Bundle savedInstanceState) {
 
         //initialize text and buttons
 
-        this.title = (TextView)findViewById(R.id.view_record_title);
-        this.date = (TextView)findViewById(R.id.view_record_date);
-        this.description = (TextView)findViewById(R.id.view_record_description);
-        this.view_front_body_button = (ImageButton)findViewById(R.id.view_front_body);
```

Comment:
you could change it back to 
this.view_front_body_button = (ImageButton)findViewById(R.id.view_front_body);
this.view_back_body_button = (ImageButton)findViewById(R.id.view_back_body);